### ISSUE TITLE
Is possible to view content in the debugger in multiple lines?

### ISSUE THREAD

**Issue by nath3an** [zero-plusplus/vscode-autohotkey-debug#211]
```ahk
Text =
(
   a
   a
   a
   a
)
x:= {}
x[1] := Text
```

Display as:
![image](https://user-images.githubusercontent.com/105518872/168377263-167f1c2e-0603-44b6-aef5-50f582f51553.png)

I wonder ifs possible to display it like this:
![image](https://user-images.githubusercontent.com/105518872/168377344-198b40c2-4a68-4c5f-a804-c0027ede332b.png)

Not by default, I mean, maybe an option?

**Comment by zero-plusplus** [comment]
I do not know how your second image was created. I think the data inspections and hover information were not designed to display multiple rows.

---

If you want to see the contents of the string, use `View as Text` in the [context menu of the data inspection](https://github.com/zero-plusplus/vscode-autohotkey-debug/wiki/Data-inspection#context-menu).

**Comment by nath3an** [comment]
> The following useful actions are available in the menu that appears when you right-click on a variable.
View Value - Display the variable value displayed for data inspection to new editor
Copy Value - Copy the variable value displayed for data inspection
Copy as Expression - Copy the variable name path
Add to Watch - Add the selected variable to the [watch expression](https://github.com/zero-plusplus/vscode-autohotkey-debug/wiki/Watch-expression)

How can I activate it? i don't see any of these options when I right-click the variable in the script or in the watch window.

**Comment by zero-plusplus** [comment]
The feature is only available in [Data Inspection](https://code.visualstudio.com/docs/editor/debugging#_data-inspection).

**Comment by nath3an** [comment]
It shows the same way in data Inspection 
```
`a`n    a`n      `n`
```
Is not possible to replace these 
```
`n
```
for new lines in the debugger watch?

**Comment by zero-plusplus** [comment]
Data inspection, watch expression, and hover information cannot display multiple lines string.

There are several reasons not to implement this, but when I implemented it in the first place, the vscode specification did not allow for multi-line display.

Therefore, I was curious how your second image was created.

The `View as Text` I proposed is a feature I implemented to work around the above limitation.
